iroha-cli/interactive/impl/interactive_transaction_cli.cpp

```diff
@@ -418,6 +422,15 @@ namespace iroha_cli {
           src_account_id, dest_account_id, asset_id, amount);
     }
 
+    std::shared_ptr<iroha::model::Command>
+    InteractiveTransactionCli::parseSetAccountDetail(
+        std::vector<std::string> params) {
+      auto account_id = params[0];
+      auto key = params[1];
+      auto value = params[2];
+      return std::make_shared<SetAccountDetail>(account_id, key, value);
```

Comment:
what if `params` has no 3 elements? Let's say, 2.
Is it possible?

Reply:
No, cli would not get here if there were other than 3 elements